cmd/opentelemetry-collector/app/defaults/default_config_test.go

```diff
@@ -19,433 +19,154 @@ import (
-			assert.Equal(t, test.receivers, recvs)
-		})
-	}
-}
-
-func TestDefaultAgentConfig(t *testing.T) {
-	tests := []struct {
-		config  map[string]interface{}
-		service configmodels.Service
-	}{
-		{
-			config: map[string]interface{}{"resource.labels": "foo=bar"},
 			service: configmodels.Service{
 				Extensions: []string{"health_check"},
 				Pipelines: configmodels.Pipelines{
 					"traces": &configmodels.Pipeline{
 						InputType:  configmodels.TracesDataType,
 						Receivers:  []string{"jaeger"},
 						Processors: []string{"resource"},
-						Exporters:  []string{"jaeger"},
+						Exporters:  []string{"jaeger_elasticsearch", "jaeger_kafka", "jaeger_memory"},
 					},
 				},
 			},
 		},
 		{
+			cfg: ComponentSettings{
+				ComponentType: Ingester,
+				StorageType:   "elasticsearch",
+			},
 			service: configmodels.Service{
 				Extensions: []string{"health_check"},
 				Pipelines: configmodels.Pipelines{
 					"traces": &configmodels.Pipeline{
 						InputType: configmodels.TracesDataType,
-						Receivers: []string{"jaeger"},
-						Exporters: []string{"jaeger"},
-					},
-				},
-			},
-		},
-	}
-	for _, test := range tests {
-		t.Run(fmt.Sprintf("%v", test.config), func(t *testing.T) {
-			v, _ := jConfig.Viperize(app.AddComponentFlags)
-			for key, val := range test.config {
-				v.Set(key, val)
-			}
-			factories := Components(v)
-			cfg := AgentConfig(factories)
-			require.NoError(t, config.ValidateConfig(cfg, zap.NewNop()))
-
-			assert.Equal(t, test.service, cfg.Service)
-			assert.Equal(t, 1, len(cfg.Receivers))
-			assert.IsType(t, &jaegerreceiver.Config{}, cfg.Receivers["jaeger"])
-			assert.Equal(t, 1, len(cfg.Exporters))
-			assert.IsType(t, &jaegerexporter.Config{}, cfg.Exporters["jaeger"])
-			processorMap := map[string]bool{}
-			for _, p := range test.service.Pipelines["traces"].Processors {
-				processorMap[p] = true
-			}
-			if processorMap["resource"] {
-				assert.Equal(t, len(processorMap), len(cfg.Processors))
-				assert.IsType(t, &resourceprocessor.Config{}, cfg.Processors["resource"])
-			}
-		})
-	}
-}
-
-func TestCreateAgentReceivers(t *testing.T) {
-	tests := []struct {
-		args      []string
-		receivers configmodels.Receivers
-	}{
-		{
-			args: []string{""},
-			receivers: configmodels.Receivers{
-				"jaeger": &jaegerreceiver.Config{
-					TypeVal: "jaeger",
-					NameVal: "jaeger",
-					Protocols: map[string]*receiver.SecureReceiverSettings{
-						"thrift_compact": {
-							ReceiverSettings: configmodels.ReceiverSettings{
-								Endpoint: udpThriftCompactEndpoint,
-							},
-						},
-						"thrift_binary": {
-							ReceiverSettings: configmodels.ReceiverSettings{
-								Endpoint: udpThriftBinaryEndpoint,
-							},
-						},
+						Receivers: []string{kafkaRec.TypeStr},
+						Exporters: []string{elasticsearch.TypeStr},
 					},
 				},
 			},
 		},
 		{
-			args: []string{
-				"--processor.jaeger-binary.server-host-port=host:1",
```

Comment:
Looks like this test has been removed? I think this is still required.

Reply:
The code is fully covered in this class. The same test is in `jaeger_receiver_test.go`.